.github/workflows/smoke-test-release.yml

```diff
@@ -1,14 +1,14 @@
 name: Smoke test release
 on:
     release:
-        types: [published]
+        types: [released, prereleased]
     workflow_dispatch:
         inputs:
             tag:
                 description: 'WooCommerce Release Tag'
                 required: true
 concurrency:
-    group: ${{ github.workflow }}-${{ github.ref }}
+    group: ${{ github.workflow }}-${{ github.event.release.tag_name || inputs.tag }}
```

Comment:
[Fix]

I noticed that the concurrency group was erroneous because the main intention of this workflow is to group the release smoke tests based on the release version (tag) they're testing, not based on `github.ref`. Leaving the concurrency group to use `github.ref` will cause a smoke test for release `7.5.0`, for example, to unexpectedly cancel the smoke test of another release version.

I fixed it here so that the release smoke tests are grouped by release tag, (whether this tag came from the `release` or `workflow_dispatch` event) thereby preventing concurrent release smoke tests cancelling each other.